projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_configuration_plugin.py

```diff
@@ -29,6 +29,18 @@ def vdk_configure(self, config_builder: ConfigurationBuilder) -> None:
             default_value=None,
             description="Default Ingestion target to be used.",
         )
+        config_builder.add(
+            key="INGEST_PLUGIN_EXECUTION_SEQUENCE",
+            default_value=None,
+            description="""A string of coma-separated ingestion plugin method names,
+            indicating the sequence in which the ingestion plugins would process the
+            payload. For example:
+                   INGEST_PLUGIN_EXECUTION_SEQUENCE="pre-ingest-process,ingest-http"
```

Comment:
Also mention how does it work with ingest method configuration. 

Also why execution - but otherwise you are mentioning process? Would this mean we can send data to multiple locations ?

Reply:
Added additional information.